parallelstone_manager/core/dependencies.py

```
@@ -16,24 +16,38 @@ async def get_client(self) -> RCONClient:
         client = await self.get_client()
+        if not await client.is_connected():
+            print("RCON connection lost. Reconnecting...")
+            await self.connect()
+
         try:
-            await client.connect()
             result = await client.send_command(command)
             return result
-        finally:
-            await client.close()
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)